etc/legacy/index.html

```diff
@@ -222,6 +222,12 @@ <h3 class="logo-header">Provide your email to receive news about Crowds Cure Can
               </svg>
               <a>Liver</a>
             </div>
+			 <div class="tool" data-command="setWLPresetBrain">
```

Comment:
You can remove these changes. We aren't using this file at all anymore. It's in the legacy folder because it is the old version of the site.

Reply:
Removed the changes from "etc/legacy/index.html"